lib/cli/commands/unban.ts

```diff
@@ -2,17 +2,17 @@ import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
 import { UnbanRequest } from '../../proto/xudrpc_pb';
 
-export const command = 'unban <node_pub_key> [reconnect]';
+export const command = 'unban <node_key> [reconnect]';
 
-export const describe = 'unban an xud node';
+export const describe = 'unban a previously banned peer';
 
 export const builder = {
-  node_pub_key: {
-    description: 'NodePubKey of the node to unban',
+  node_key: {
```

Comment:
Same here. Think we also need to change:
`request.setNodePubKey(argv.node_pub_key);`

Reply:
Fixed.